source/App/source/Common.Tests/Fixtures/B2CClientAppSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Energinet.DataHub.Core.App.Common.Tests.Fixtures
+{
+    /// <summary>
+    /// Name and credential settings for a B2C 'test client app'.
```

Comment:
'test client app' is mentioned in a couple of comments, but does not appear in any naming or so. Should we keep as is or perhaps alter the comments like  
```suggestion
    /// Name and credential settings for a B2C client app to be used for testing.
```